packages/list/src/lib.rs

```diff
@@ -2,8 +2,24 @@ use std::str::FromStr;
 
 use serde_json::{json, Value};
 
+macro_rules! module {
+    ($name:expr, $data:expr $(,$($args:tt)*)?) => {json!({"name": $name $(,"arguments":$($args)*)*, "data": $data})}
+}
+
 macro_rules! import {
-    ($e:expr) => {json!({"name": "set-add", "arguments": {"name": "imports"}, "data": $e})}
+    ($package:expr) => {module!("set-add", $package, {"name": "imports"})}
+}
+
+macro_rules! raw {
```

Comment:
Can't we just use json string literals instead of the raw macro

Reply:
Yes, we can! I just refactored the code as it was without changing it, but I will change to the raw macro before merging